public/media_engagement/media_engagement.js

```diff
@@ -34,335 +35,3 @@ firebase.auth().onAuthStateChanged(function (user) {
         window.location.replace('auth.html')
     }
 });
-
```

Comment:
This PR appears to be deleting a lot of functionality as well as changing to production db. Can you confirm that is intended?

Reply:
Yes thats intention. It was easier that way due to the current dashboard design.

Reply:
ok thanks, in general that should probably have gone into a separate PR as it's a different logical change

Reply:
Great got it.